apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/VedtaksbrevService.kt

```diff
@@ -112,7 +113,7 @@ class VedtaksbrevService(
 
         val payload = when (prosessType) {
             AUTOMATISK -> null
-            MANUELL -> SlateHelper.hentInitiellPayload(behandling.sakType, behandling.vedtak.type)
+            MANUELL -> SlateHelper.hentInitiellPayload(behandling.sakType, behandling.vedtak.type, behandling)
```

Comment:
Kunne fjernet de første to parameterene når du allikevel sender inn hele behandlingsobjektet.